rules/mastg-android-unvalidated-deeplink-data.yml

```
@@ -0,0 +1,15 @@
+rules:
+- id: android-unvalidated-deeplink-data
+  languages:
+    - java
+  severity: WARNING
```

The severity should come above the language attribute
```suggestion
  severity: WARNING
  languages:
    - java
```